components/FormikFields/PaperInputPicker/index.js

```diff
@@ -62,6 +65,34 @@ const PaperInputPicker = ({
   const [pictureUris, setPictureUris] = React.useState({});
   const [image, setImage] = React.useState(null);
 
+  const [numberedQestions, setNumberedQuestions] = React.useState({});
+  const [questionsToRepeat, setQuestionsToRepeat] = React.useState([]);
+  const [additionalQuestions, setAdditionalQuestions] = React.useState([]);
+  const [individualLoopsAdded, setIndividualLoopsAdded] = React.useState(0);
```

Comment:
These should be the only "state-related" stuff that stay in this file (since it seems like you need them for the `PaperInputPicker` inside the Looper field type here , we can do all the rest of the state management (add loop and remove loop in `Looper.js`

this will save us a lot of lines of code as this mega of a component `<PaperInputerPicker/>` gets more inputs